todo_app/trello_items.py

```diff
@@ -0,0 +1,42 @@
+from flask import trello
+import requests
+import json
+
+
+class Item:
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list['name'])
+
+
+def add_items():
+    
+    return trello.get('items')
```

Comment:
See my comment below on adding a "get_items" function - we want this function to do something similar rather than relying on an existing trello module (which doesn't currently exist!)